keps/sig-network/3705-cloud-node-ips/README.md

```diff
@@ -266,9 +296,31 @@ Notes:
     dual-stack IPv4-primary configuration. (In particular, you would
     have IPv4-primary nodes but IPv6-primary pods.)
 
-  - `--node-ip 1.2.3.4,abcd::ef01` was previous valid syntax when
+  - `--node-ip 1.2.3.4,abcd::ef01` was previously valid syntax when
     using no `--cloud-provider`, but was not valid for cloud kubelets.
 
+<<[UNRESOLVED multiple-addresses ]>>
+
+In the examples above, `--node-ip IPv4` means "pick any one IPv4
+address", not "use all IPv4 addresses". This seemed to me to be
+consistent with the behavior when specifying a specific IP address (in
+which case all other IPs of the same NodeAddressType are removed from
+the list), but is inconsistent with the behavior of `--node-ip
+0.0.0.0` / `--node-ip ::` with legacy cloud providers and bare metal
+(where that affects the _sorting_ of IPs but does not do any
+_filtering_).
+
+In past discussions (eg https://kep.k8s.io/1664,
+https://issues.k8s.io/95768) no one has ever been entirely sure what
+the point of having multiple node.status.addresses values is, and
+there has never been a way to get multiple node.status.addresses
+values (of the same IP family) on "bare metal", so that would tend to
+prevent people from designing features that depended on multiple node
+addresses. So it's not clear that there's really any downside in
+filtering out the "unused" node IPs...
```

Comment:
How/when will we close this?

Maybe to-be-resolved before gate moves to beta (or GA?), based on feedback?

Reply:
sure, i'll add that to the graduation criteria